controllers/archived.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const config = require('config');
+const express = require('express');
+const { flatMap } = require('lodash');
+
+const router = express.Router();
+
+const metrics = require('../modules/metrics');
+const { makeWelsh } = require('../modules/urls');
+const cached = require('../middleware/cached');
+
+/**
+ * Archived Routes
+ * Paths in this array will be redirected to the National Archives.
+ * We show an interstitial page a) to let people know the page has been archived
+ * and b) to allow us to record the redirect as a pageview using standard analytics behaviour.
+ */
+// prettier-ignore
+flatMap([
+    '/about-big/10-big-lottery-fund-facts',
+    '/funding/funding-guidance/applying-for-funding/*',
+    '/global-content/programmes/england/building-better-opportunities/building-better-opportunities-qa*'
+], urlPath => [urlPath, makeWelsh(urlPath)]).forEach(urlPath => {
+    router.get(urlPath, cached.noCache, function(req, res) {
+        const fullUrl = `https://${config.get('siteDomain')}${req.originalUrl}`;
```

Comment:
This `siteDomain` variable will probably change to be the new one soon, but for archive purposes we always want this to be biglotteryfund.org.uk so it would be good to hard-coded this here.

Reply:
(I know you're just moving this code)

Reply:
Yeah good call, will tweak.